src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,112 @@
+---
+linkTitle: P1 Incident Process
+title: P1 incident process
+description: The process used by the Giant Swarm support team when a priority one incident is called.
+weight: 100
+menu:
+  main:
+    parent: overview
+    weight: 20
+user_questions:
+  - What process does Giant Swarm follow in case of critical incidents?
+owner:
+  - https://github.com/orgs/giantswarm/teams/teddyfriends
+last_review_date: 2023-05-15
+---
+
+After years of handling critical enterprise workloads in production, we have hardened our incident process based on our learnings. In this document we focus on critical incidents, called `Priority 1` (P1) incidents, though some of the steps could be part of regular incidents too. 
+
+Giant Swarm classifies incidents as being either critical (`P1` or routine (`P2`). Critical incidents are those which impair a customer production system. Routine incidents are all other regular incidents which don't impact production and which have a straightforward process.
+
+## Separation of Responsibilities
+
+It’s important to make sure that everybody involved in an incident knows their role, what is expected from them, and that it doesn’t conflict with others' responsibilities. Somewhat counterintuitively, a clear separation of responsibilities allows individuals more autonomy, as they do not need to ask all the time and coordinate actions.
+
+### Roles
+
+At Giant Swarm we have two defined roles: Incident Coordinators and Operations Engineers.
+
+### Incident Coordinator
+
+The Incident Coordinator (IC) holds the high-level state about the incident. They structure the incident response, assigning responsibilities according to need and priority. De facto, the IC holds all positions/responsibilities that they have not delegated. If appropriate, they can remove roadblocks that prevent Operations Engineers from working most effectively.
+
+The IC is the public face of our incident response. It is within the ICs duty to issue periodic updates to the team involved - both customer teams and within Giant Swarm - to act as the bridge between the customer and the team. The IC will need to be in the war rooms of our customers. For that reason, we have created a [Opsgenie team](https://support.atlassian.com/opsgenie/docs/what-are-teams-in-opsgenie/) grouping all members that can act as Incident Coordinators. 
+
+If there is a dedicated IC, the IC isn’t debugging systems and keeps the focus on coordinating the team to do so, while managing the customer communication.
+
+### Operations Engineer
+
+The Operations Engineer (OE) works with the Incident Coordinator to respond to the incident and is the one responsible for debugging and applying changes to a system. 
+
+In case of Operations Engineers, we have different [Opsgenie teams](https://support.atlassian.com/opsgenie/docs/what-are-teams-in-opsgenie/) sliced by area that are oncall in a given day.
```

Comment:
this is arguably somewhat more complicated - we have both teams during the day, and areas at night

Reply:
yeah I simplify it on purpose. Anyway I am open for suggestions if you would like to describe it further